scripts/entity/components/input_components/player_input_component.gd

```diff
@@ -2,9 +2,9 @@ class_name PlayerInputComponent extends BaseInputComponent
 ## An extension to the [BaseInputComponent] class which takes in input from a player input device.
 
 
-const JUMP_INPUT_NAME = "jump"
-const LEFT_INPUT_NAME = "left"
-const RIGHT_INPUT_NAME = "right"
+const JUMP_INPUT_NAME: String = "jump"
+const LEFT_INPUT_NAME: String  = "left"
+const RIGHT_INPUT_NAME: String = "right"
```

Comment:
If gonna define, maybe should be defined as StringNames https://docs.godotengine.org/en/stable/classes/class_stringname.html

Reply:
changed it